OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/AdminControllerTests.cs

```diff
@@ -192,33 +191,18 @@ public async Task GetApplications_WhenCollectionIsEmpty_ShouldReturnNoContent()
         // Arrange
         controller.ControllerContext.HttpContext = fakeHttpContext;
         controller.ControllerContext.HttpContext.SetContextUser(Role.TechAdmin);
-        sensitiveApplicationService.Setup(s => s.GetAll(It.IsAny<ApplicationFilter>())).ReturnsAsync(new SearchResult<ApplicationDto>()
+        sensitiveApplicationService.Setup(s => s.GetAll(It.IsAny<ApplicationFilter>())).ReturnsAsync(new SearchResult<ApplicationDto>
         {
-            Entities = new List<ApplicationDto>(),
-            TotalAmount = 0,
+            Entities = applications,
```

Comment:
(Test unclear) `applications` collection seems to be not empty here. Test name mention empty collection. Please fix